client/src/routes.js

```diff
@@ -64,20 +56,6 @@ export default (
       path="/manager-resources"
       getComponent={(location, callback) => callback(null, ManagerResources)}
     />
-    <Route
-      path="/about"
-      getComponent={(location, callback) => callback(null, About)}
-    />
-    <Route
-      path="/error"
-      status={404}
-      getComponent={(location, callback) => callback(null, NotFound)}
-    />
-    {/* Catch all route */}
-    <Route
-      path="*"
-      status={404}
-      onEnter={(nextState, replace) => replace('/error')}
-    />
+    <Route path="/about" getComponent={(location, callback) => callback(null, About)} />
```

Comment:
I think we'll want the 404 /error and catch all route still, or is there somewhere else they're included?